pallets/contracts/src/benchmarking.rs

```diff
@@ -0,0 +1,245 @@
+// This file is part of the Polymesh distribution (https://github.com/PolymathNetwork/Polymesh).
+// Copyright (c) 2020 Polymath
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, version 3.
+
+// This program is distributed in the hope that it will be useful, but
+// WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+// General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+#![cfg(feature = "runtime-benchmarks")]
+
+use crate::*;
+use frame_benchmarking::benchmarks;
+use frame_support::storage::IterableStorageMap;
+use frame_system::RawOrigin;
+use pallet_contracts::PristineCode;
+use pallet_identity::benchmarking::{User, UserBuilder};
+use parity_wasm::elements::FuncBody;
+use polymesh_primitives::{
+    MetaDescription, MetaUrl, MetaVersion, SmartExtensionType, TemplateMetadata,
+};
+use sp_runtime::traits::Hash;
+
+type BaseContracts<T> = pallet_contracts::Module<T>;
+type System<T> = frame_system::Module<T>;
+
+const SEED: u32 = 0;
+const MAX_URL_LENGTH: u32 = 400;
+const MAX_DESCRIPTION_LENGTH: u32 = 100;
```

Comment:
Are these ensured by the runtime?

Reply:
No there is no maximum length check in the runtime, Right now it is a free length vector. I will add a check and later on we will assign the right numbers once we have proper benchmarks figure.

Reply:
Either these variables should be removed or should much higher limits in the benchmark (like 10_000). the benchmarking cli won't be able to calculate their effect otherwise. 

As far as limiting them in the runtime goes, we don't need that. These do not really have much of an impact.